app/models/API1811/DocumentDetailsNew.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.API1811
+
+import play.api.libs.functional.syntax.toFunctionalBuilderOps
+import play.api.libs.json.{JsPath, Reads}
+
+case class DocumentDetailsNew(chargeReferenceNumber: Option[String],
```

Comment:
The idea here is to replace our old models instead of just making additional ones, e.g. replace `DocumentDetails` with this rather than making a second class.

With that in mind, looks like you need to remove `SubItem` and `Transaction` (although please keep the charge type writes in `Transaction` - move it over to another model even if unsure of the best placement for now).

`FinancialTransactions` needs to be updated so has a sequence of this model.

When this is done you should be able to IT test your work too, to confirm the parser/connector are reading EIS JSON to models correctly. There will be some test failures involving the writes as this is now much shorter/nonexistent. Correct or remove tests as necessary (any writes tests can be re-added in the other subtask).